### DISCUSSION TITLE
Zooming with a touchpad

### DISCUSSION THREAD

**Discussion by drfritz142** [Stellarium/stellarium#2442] [Q&A] [answered]
Hi everyone, 

I occasionally use Stellarium on a laptop with a touchpad but w/o mouse. While nearly every other application known to me uses a finger pinching motion to zoom in or out, in Stellarium this advances the clock minutes. To zoom one has to instead swipe two fingers up or down. I find this irritating, and it gets me every single time. I was wondering whether there is a setting somewhere where I can assign the pinching motion to a zooming action (and perhaps assign the two-finger swipe to advancing the clock or drop it alltogether)?

Clear skies,
Andreas

**Comment by gzotti** [comment, marked as answer]
You cannot reconfigure touchpad actions, but just get used to the equivalent keyboard keys which have existed since the time when touchpads had one-finger operation only.

**Reply by drfritz142** [reply]
Thanks for the quick reply. It’s not easy for me to find the keyboard keys at night when it’s dark. The touchpad is a lot easier for me, also because my fingers are already on it for moving around in the night sky. I’ll probably  just have to get used to it. Or I’ll fork the repo and try to build it my way… 😜 

BTW: Is there a section in the user guide where touchpad actions are explained?

best, andreas
